satpy/etc/readers/aws_l1b_nc.yaml

```
@@ -0,0 +1,535 @@
+reader:
+  name: aws_l1b_nc
+  short_name: AWS L1B RAD NetCDF4
+  long_name: AWS L1B Radiance (NetCDF4)
+  description: Reader for the ESA AWS (Arctic Weather Satellite) Sounder level-1b files in netCDF4.
+  reader: !!python/name:satpy.readers.yaml_reader.FileYAMLReader
+  sensors: [aws,]
```

```suggestion
  sensors: [mws,]
```

Not sure if it should be a list or just a string :thinking: